Configuration/TCA/Overrides/tx_telephonedirectory_domain_model_building.php

```diff
@@ -14,20 +14,22 @@
 use JWeiland\Maps2\Tca\Maps2Registry;
 use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
 
-// Add tx_maps2_uid column to telephone directory table
-if (ExtensionManagementUtility::isLoaded('maps2')) {
-    Maps2Registry::getInstance()->add(
-        'telephonedirectory',
-        'tx_telephonedirectory_domain_model_building',
-        [
-            'addressColumns' => ['street', 'house_number', 'zip', 'city'],
-            'defaultCountry' => 'Germany',
-            'synchronizeColumns' => [
-                [
-                    'foreignColumnName' => 'title',
-                    'poiCollectionColumnName' => 'title',
+call_user_func(function () {
```

Comment:
Please check for TYPO3 and PHP version, if we can use this one here:

```suggestion
call_user_func(static function (): void {
```